ci-operator/step-registry/aws/provision/public-to-private/aws-provision-public-to-private-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: aws-provision-public-to-private
```

Comment:
I'm not sure, but I think `aws-cluster-public-to-private` or `aws-public-to-private` is better than `aws-provision-`

Reply:
agree.